pkg/kubernetes/client/apply.go

```diff
@@ -31,6 +31,10 @@ func (k Kubectl) apply(data manifest.List, opts ApplyOpts) error {
 		argv = append(argv, "--force")
 	}
 
+	if !opts.Validate {
+		argv = append(argv, "--validate=false")
+	}
```

Comment:
As there are now two of this, it might be better two use an empty `switch`:

```go
switch {
  case opts.Force:
    argv = append(argv, "--force")
  case !opts.Validate:
    argv = append(argv, "--validate=false")
}
```

Reply:
Are this options anyhow correlated? we will break out of switch if first statement is true (so we cannot force and disable validation)

Reply:
true, forgot this is golang, we need `fallthrough` as well .... 

Taking this into consideration, it becomes equally verbose, so there is no point in doing this.